.cargo/audit.toml

```diff
@@ -4,7 +4,8 @@ ignore = [
   "RUSTSEC-2024-0384", # 'instant' dependency in 'futures-lite' [unmaintained]
   "RUSTSEC-2024-0436", # 'paste' dependency in 'libp2p' [unmaintained]
   "RUSTSEC-2023-0071", # 'rsa' dependency in 'sqlx' [unfixed]
-  "RUSTSEC-2024-0388", # 'derivative' in 'alloy' [unmaintained]
+  "RUSTSEC-2024-0388", # 'derivative' in 'alloy' [unmaintained],
+  "RUSTSEC-2025-0052", # 'async-std' in 'async-tar' [unmaintained]
```

Comment:
Can we rather disable the async-std by a feature in that dependency? It would be preferable to minimize it that way, since we only need the tokio part.

Reply:
It cannot be disabled, we'd need to replace the `async-tar` dep completely.